py/legacypipe/reference.py

```diff
@@ -161,9 +167,29 @@ def read_gaia(targetwcs):
     # [decam-data 2770] Re: [desi-milkyway 639] GAIA in DECaLS DR7
     # But shifted one mag to the right in G.
     gaia.G = gaia.phot_g_mean_mag
+
+    # Gaia to DECam color transformations for stars
+    color = gaia.phot_bp_mean_mag - gaia.phot_rp_mean_mag
+    color[np.logical_not(np.isfinite(color))] = 0.
+    color = np.clip(color, -0.5, 3.3)
+    # Use Arjun's Gaia-to-DECam transformations.
+    for b,coeffs in [
+        ('g', [-0.11368, 0.37504, 0.17344, -0.08107, 0.28088,
+           -0.21250, 0.05773,-0.00525]),
+        ('r', [ 0.10533,-0.22975, 0.06257,-0.24142, 0.24441,
+            -0.07248, 0.00676]),
+        ('z', [ 0.46744,-0.95143, 0.19729,-0.08810, 0.01566])]:
+        mag = gaia.G.copy()
+        for order,c in enumerate(coeffs):
+            mag += c * color**order
+        gaia.set('decam_mag_%s' % b, mag)
+
+    AEN = gaia.astrometric_excess_noise
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [local variable 'AEN' is assigned to but never used](https://app.codacy.com/app/dstndstn/legacypipe/pullRequest?prid=4730106)